terraform/main.tf

```diff
@@ -59,14 +59,21 @@ resource "aws_instance" "ec2_instance" {
   tags = {
     Name = "Linc-CV-${terraform.workspace}-${formatdate("YYYY-MM-DD-hh:mm", timestamp() )}"
   }
-    user_data = terraform.workspace == "prod" ? file("startup_script.sh") : file("startup_script_staging.sh")
+    user_data = terraform.workspace == "staging" ? file("startup_script_staging.sh") : file("startup_script.sh")
     user_data_replace_on_change = true
 }
 
-resource "aws_eip_association" "eip_assoc" {
-  count = terraform.workspace == "prod" ? 1 : 0
+resource "aws_eip_association" "eip_assoc_green" {
+  count = terraform.workspace == "green" ? 1 : 0
```

Comment:
What does the count do?

Reply:
The count basically means if the workspace is "green" deploy this EIP